tests/functional_tests/tests/test_atclient_get_atkeys.c

```diff
@@ -146,6 +146,7 @@ exit: {
 
 static int test_4_atclient_get_atkeys_null_regex(atclient *ctx, const char *scan_regex, const bool showhidden) {
   int ret = 1;
+  return 0; // NOTE: disabled temporarily DONT LET THIS GET MERGED
```

Comment:
I'll make PR draft until this is addressed

Reply:
Addressed in [40f6221](https://github.com/atsign-foundation/at_c/pull/475/commits/40f6221f743d7536dd9dd3fa102657fb0bc8230f)